### PR TITLE
[18.0][MIG] base_edi

### PR DESCRIPTION
**Pull request by JordiMForgeFlow**

### CODE REVIEW

**Comment by JordiMForgeFlow** [comment]
It seems CI fails because there are no tests?

CC @etobella @simahawk

**Comment by simahawk** [comment]
you can add a silly test to check that the module category is created for instance (separate commit that we can backport)

**Comment by JordiMForgeFlow** [comment]
@simahawk done! :)

File: base_edi/tests/test_base_edi.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from odoo.tests import common
+
+
+class TestBaseEDI(common.TransactionCase):
+    """Simple test for the CI"""
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```
Comment (simahawk) [inline base_edi/tests/test_base_edi.py]:
```suggestion
```

File: base_edi/tests/test_base_edi.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 ForgeFlow
+# @author: Jordi Masvidal <jordi.masvidal@forgeflow.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo.tests import common
+
+
+class TestBaseEDI(common.TransactionCase):
```
Comment (simahawk) [inline base_edi/tests/test_base_edi.py:8]: can you please move this change to its own commit (as per my initial comment)?

**Comment by JordiMForgeFlow** [comment]
@simahawk sorry, I forgot to keep it separated. I have now corrected it :)

**Review by simahawk — APPROVED** [review]
